metadata-ingestion/tests/integration/lookml/test_lookml.py

```diff
@@ -887,3 +887,37 @@ def test_duplicate_field_ingest(pytestconfig, tmp_path, mock_time):
         output_path=tmp_path / mce_out_file,
         golden_path=golden_path,
     )
+
+
+@freeze_time(FROZEN_TIME)
+def test_view_to_view_lineage_and_liquid_template(pytestconfig, tmp_path, mock_time):
+    test_resources_dir = pytestconfig.rootpath / "tests/integration/lookml"
+    mce_out_file = "vv_lineage_liquid_template_golden.json"
+
+    new_recipe = get_default_recipe(
+        f"{tmp_path}/{mce_out_file}",
+        f"{test_resources_dir}/vv-lineage-and-liquid-templates",
+    )
+
+    new_recipe["source"]["config"]["liquid_variable"] = {
+        "_user_attributes": {
+            "looker_env": "dev",
+            "dev_database_prefix": "employee",
+            "dev_schema_prefix": "public",
+        },
+        "dw_eff_dt_date": {
+            "_is_selected": True,
```

Comment:
for certain "looker native" variables like `view_field._is_selected` - we should have good defaults that don't require the end user to manually configure things. In this case, any time we see this `_is_selected` format we should just replace it with true unless explicitly overridden by the user

there's likely other similar native looker variables - please make sure we've accounted for all of them

Reply:
done